bin/stacks

```diff
@@ -0,0 +1,333 @@
+    k.set_contents_from_string(tpl)
+    url = k.generate_url(expires_in=30)
+
+    try:
+        if update:
+            conn.update_stack(name, template_url=url, tags=tags, capabilities=['CAPABILITY_IAM'])
+        else:
+            conn.create_stack(name, template_url=url, tags=tags, capabilities=['CAPABILITY_IAM'])
+    except BotoServerError as err:
+        print(err)
+
+
+def delete_stack(conn, name, region, profile):
+    '''Deletes stack given its name'''
+    msg = ('You are about to delete the following stack:\n'
+           'name: {}\n'
+           'region: {}\n'
+           'profile: {}\n').format(name, region, profile)
+    print(msg)
+    response = input('Are you sure? [y/N] ')
+    yes = ['y', 'Y', 'yes', 'YES', 'Yes']
+    if response in yes:
+        conn.delete_stack(name)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-p', '--profile', default=DEFAULT_PROFILE)
+    parser.add_argument('-r', '--region')
+    subparsers = parser.add_subparsers(title='available subcommands', dest='subcommand')
+
+    parser_list = subparsers.add_parser('list', help='List stacks')
+    parser_list.add_argument('-v', '--verbose', action='store_true')
+    parser_list.add_argument('name', default='*', nargs='?',
+                             help='stack name or unix shell-style pattern')
+
+    parser_create = subparsers.add_parser('create', help='Create a new stack')
+    parser_create.add_argument('-t', '--template', required=True, type=argparse.FileType())
+    parser_create.add_argument('name')
+    parser_create.add_argument('-e', '--env', required=True)
+    parser_create.add_argument('-d', '--dry-run', action='store_true')
+
+    parser_update = subparsers.add_parser('update', help='Update an existing stack')
+    parser_update.add_argument('-t', '--template', required=True, type=argparse.FileType())
+    parser_update.add_argument('name')
+    parser_update.add_argument('-e', '--env', required=True)
+    parser_update.add_argument('-d', '--dry-run', action='store_true')
+
+    parser_delete = subparsers.add_parser('delete', help='Delete an existing stack')
+    parser_delete.add_argument('name')
+
+    args = parser.parse_args()
+
+    if not args.subcommand:
+        parser.print_help()
+
+    # Set region name from cli arg or aws config file
+    if args.region:
+        region = args.region
+    elif get_region_name(args.profile):
+        region = get_region_name(args.profile)
+
+    if args.subcommand == 'list':
+        conn = boto.cloudformation.connect_to_region(region, profile_name=args.profile)
+        print(list_stacks(conn, args.name, args.verbose))
+        conn.close()
+
+    if args.subcommand == 'create' or args.subcommand == 'update':
+        ec2_conn = boto.ec2.connect_to_region(region, profile_name=args.profile)
+        vpc_conn = boto.vpc.connect_to_region(region, profile_name=args.profile)
+        cf_conn = boto.cloudformation.connect_to_region(region, profile_name=args.profile)
```

Comment:
This is repeated over all three commands, maybe move out? also should rename in the others to be `cf_conn` too.